dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -125,6 +125,8 @@
       <LinkIncremental>false</LinkIncremental>
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <!-- EnableCOMDATFolding and OptimizeReferences are set to true below for all build configurations. -->
```

Comment:
>below [](http://example.com/codeflow?start=70&length=5)

This is my mistake I think, but since you're here, can you change this to above?

Reply:
Sure, no problem. Thanks for noticing it!